23.0.0.9 post ID review (#3345)
OpenLiberty/blogs: posts/2023-09-19-23.0.0.9.adoc

:url-about: /
//Blank line here is necessary before starting the body of the post.

> You can use Spring Boot 3.x capabilities in your Liberty applications with the new Spring Boot Support 3.0 feature. You can also now authenticate your OpenID Connect clients using the Private Key method instead of client secrets, and if you use LTPA or JWT cookies, you can now use different cookies for different applications, according to their context root.

Reviewer: ```suggestion
You can use Spring Boot 3.x capabilities in your Liberty applications with the new Spring Boot Support 3.0 feature. You can also now authenticate your OpenID Connect clients using the more secure Private Key method instead of client secrets, and, if you use LTPA or JWT cookies, you can now use different cookies for different applications, according to their context root.
```

> You can use Spring Boot 3.x capabilities in your Liberty applications with the new Spring Boot Support 3.0 feature. You can also now authenticate your OpenID Connect clients using the more secure Private Key method instead of client secrets, and if you use LTPA or JWT cookies, you can now use different cookies for different applications, according to their context root.

Reviewer: ```suggestion
You can use Spring Boot 3.x capabilities in your Liberty applications with the new Spring Boot Support 3.0 feature. You can also now authenticate your OpenID Connect clients using the Private Key method, which is more secure than using client secrets, and if you use LTPA or JWT cookies, you can now use different cookies for different applications, according to their context root.
```

> You can use Spring Boot 3.x capabilities in your Liberty applications with the new Spring Boot Support 3.0 feature. You can also now authenticate your OpenID Connect clients using the more secure Private Key method instead of client secrets, and if you use LTPA or JWT cookies, you can now use different cookies for different applications, according to their context root.

Author: ```suggestion
You can use Spring Boot 3.x capabilities in your Liberty applications with the new Spring Boot Support 3.0 feature. You can also now authenticate your OpenID Connect clients using the Private Key method, which is more secure than using client secrets, and if you use LTPA or JWT cookies, you can now use different cookies for different applications, according to their context root.
```

> You can use Spring Boot 3.x capabilities in your Liberty applications with the new Spring Boot Support 3.0 feature. You can also now authenticate your OpenID Connect clients using the more secure Private Key method instead of client secrets, and, if you use LTPA or JWT cookies, you can now use different cookies for different applications, according to their context root.

Reviewer: ```suggestion
You can use Spring Boot 3.x capabilities in your Liberty applications with the new Spring Boot Support 3.0 feature. You can also now authenticate your OpenID Connect clients using the more secure Private Key method instead of client secrets, and if you use LTPA or JWT cookies, you can now use different cookies for different applications, according to their context root.
```

> And finally, happy birthday! Open Liberty is 6 years old today. See link:{url-prefix}/blog/2022/09/21/history-maker-projects.html[our history in maker projects].

Reviewer 2: This is minor, feel free to ignore it. 
```suggestion
And finally, happy birthday! Open Liberty is now 6 years old. See link:{url-prefix}/blog/2022/09/21/history-maker-projects.html[our history in maker projects].
```

[link=https://stackoverflow.com/tags/open-liberty]
image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]

// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/25973
// Contact/Reviewer: hlhoots
// // // // // // // //
[#sb3]
== Spring Boot 3.0 support

> The Liberty Spring Boot Support 3.0 feature supports running a Spring Boot 3.0 application on Liberty. It also provides the capability to link:{url-prefix}/docs/latest/deploy-spring-boot.html#thin[thin the application] when you create applications in containers.

Reviewer 2: I think I prefer the (albeit more verbose) pervious wording.  Prior to this release, the runtime did provide some Spring Boot 3 feature support, but with this delivery, the support is being expanded.
```suggestion
The Liberty Spring Boot Support 3.0 feature provides a more complete support for running Spring Boot 3.0 applications on Liberty. It also provides the capability to link:{url-prefix}/docs/latest/deploy-spring-boot.html#thin[thin the application] when you create applications in containers.
```

Author: Yes, but as stated, that's not really clear and just adds more words. That said, I'm ok with leaving it as it was
